tbx/blog/models.py

```diff
@@ -124,7 +124,10 @@ class BlogPage(ColourThemeMixin, ContactMixin, SocialFields, Page):
     listing_summary = models.TextField(blank=True)
     canonical_url = models.URLField(blank=True, max_length=255)
     related_sectors = ParentalManyToManyField(
-        "taxonomy.Sector", related_name="blog_posts"
+        "taxonomy.Sector",
+        related_name="blog_posts",
+        null=True,
```

Comment:
**Suggestion:** You don't need to set null=True on a ManyToManyField since this relationship relies on a through table, and blank=True means that we can leave the field empty an no corresponding row is added to the through table

Reply:
Thanks, TIL!

Updated in 52420c9